alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/alfrescointegration/MetricsControllerBeanPostProcessor.java

```diff
@@ -23,7 +23,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MetricsControllerBeanPostProcessor.class);
 
-    private static final String METER_REGISTRY_BEAN_ID = "meterRegistry";
+    public static final String METER_REGISTRY_BEAN_ID = "meterRegistry";
```

Comment:
Not necessary? (Also not something we should / want to expose I think?)

Reply:
Yes, not necessary anymore. I tried to get the bean by name in code in a childcontext postprocessor, but that is not possible.